libpod/runtime_img.go

```diff
@@ -95,6 +98,260 @@ type imageDecomposeStruct struct {
 	transport   string
 }
 
+func (k *KpodImage) assembleFqName() string {
+	return fmt.Sprintf("%s/%s:%s", k.Registry, k.ImageName, k.Tag)
+}
+
+func (k *KpodImage) assembleFqNameTransport() string {
+	return fmt.Sprintf("%s%s/%s:%s", k.Transport, k.Registry, k.ImageName, k.Tag)
+}
+
+//KpodImage describes basic attributes of an image
+type KpodImage struct {
```

Comment:
Let's just rename this to Image. libpod.Image matches our existing libpod.Container, libpod.Pod

Reply:
Agree

Reply:
Done